packages/hooks/src/useAllPaymentAgentList.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+import { useWS } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const useAllPaymentAgentList = () => {
+    const { client } = useStore();
+    const { residence } = client;
+    const { send, ...rest } = useWS('paymentagent_list');
+
+    useEffect(() => {
+        send({ paymentagent_list: residence });
+    }, [residence, send]);
```

Comment:
@heorhi-deriv Should we do the call in a `useEffect` here or expose a send function and let the parent component handle the call, Something similar to `useVerifyEmail` hook 🤔